crates/oxc_minifier/src/compressor/mod.rs

```diff
@@ -279,6 +292,14 @@ impl<'a, 'b> VisitMut<'a, 'b> for Compressor<'a> {
         self.compress_return_statement(stmt);
     }
 
+    fn visit_variable_declaration(&mut self, decl: &'b mut VariableDeclaration<'a>) {
+        let is_const = decl.kind.is_const(); // let, var
```

Comment:
`// let, var`? I think this can be removed.

Reply:
VariableDeclarator has a `kind` as well, for convienience.

Reply:
I didn't see that, thanks for pointing it out!